Tests/SonarScanner.MSBuild.Shim.Test/SonarEngineWrapperTest.cs

```diff
@@ -127,6 +127,66 @@ public void FindJavaExe_JavaHomeSet_DoesNotExist()
             $"Could not find Java, falling back to using PATH: {context.JavaFileName}");
     }
 
+    [TestMethod]
+    public void Execute_ScannerOptsFromEnv_UsedForJavaInvocation()
+    {
+        using var scope = new EnvironmentVariableScope().SetVariable(EnvironmentVariables.SonarScannerOptsVariableName, "-DJavaParam=Env");
```

Comment:
Add a test with more than one parameter like e.g. needed in this scenario:
https://sionwilliams.com/posts/2019-04-25_sonar_scanner_certs/